README.md

```diff
@@ -1,50 +1,74 @@
+- [ ] Ensure Release practices are documented.
 - [ ] Add at least one of the adopted-ember-addons org owners (`melsumner`, `alonbukai`, or `knownasilya`) to the maintainers of the package on `npm`. Remove other maintainers.
 
 ## Adopting an addon
-Anyone can submit a pull request to help maintain an addon in this repo!
+
+Anyone can submit a pull request to help maintain an addon in this repository!
 
 It's also possible that you'd like to wear more of an official maintainer hat, and that's cool too! If you want to help maintain one of these addons:
-- please open an issue that requests to be added as a contributor or ping melsumner on Ember Discord
-- make sure you have 2FA set up for GitHub
+
+- Please open an issue that requests to be added as a contributor or ping `@Melanie#1618` on Ember Discord.
+- Make sure you have two-factor authentication (2FA) set up and enabled for GitHub.
 
 Once we've had a chat, we can add you as a contributor. Easy-peasy!
 
 ## Standardizing Addon Maintenance
 
-By standardizing some aspects of maintaining addons we can ensure a consistent experience for contributors and maintainers.
+Adopted Ember Addons is a community effort. Maintaining the addons requires people with different backgrounds, experiences and opinions to work together successfully. Setting shared conventions for some aspects of addons maintenance provides a reference point how common problems should be solved within the org. This ensures a consistent experience for contributors and maintainers across the different addons within the org. It also reduces the entry barrier for new contributors and helps them getting started by providing clear guidance. Additionally it reduces the risk of [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality) by providing a dedicated place to discuss and change patterns and practices.
+
+### Terminology
+
+This document uses the keywords *must*, *must not*, *should*, *should not* and *may*.
+
+*Must* and *must not* express absolute requirements. Any addon within the org that do not follow these requirements should be updated as soon as possible. A pull request that changes an addon within the org accordingly to these requirements must be merged.
+
+*Should* and *should not* express a recommendation. There may be valid reasons to not follow the recommendation in some edge cases. If so the reasons should be documented.
+
+*May* means that an item is truly optional. It's an individual decision for each addon within the org to implement this item or not. A given decision should not be questioned to avoid changing it back and forth. 
+
+> This terminology is based on [RFC 2119](https://tools.ietf.org/html/rfc2119), which is used by many specification documents.
+
+### Changelog
+
+All addons within the org should have a changelog. The changelog may not cover versions that were released before it was introduced.
```

Comment:
Adding a changelog wasn't a requirement nor a clear recommendation before. But I think at least a recommendation fits better than a *may* for this case. I explicitly added that it may not cover historical versions to reduce the initial work for adding a changelog.